src/backend/lima.ts

```diff
@@ -2021,7 +2023,9 @@ class LimaKubernetesBackend extends events.EventEmitter implements K8s.Kubernete
   }
 
   async stop() {
-    await this.vm.execCommand({ root: true }, '/sbin/rc-service', '--ifstarted', 'k3s', 'stop');
+    if (this.cfg?.enabled) {
+      await this.vm.execCommand({ root: true }, '/sbin/rc-service', '--ifstarted', 'k3s', 'stop');
+    }
```

Comment:
Rather than checking if it's enabled, we should instead check if the VM is _running_, and only do this if it is.  If the VM is off, we should skip this even if we wanted k3s to be running.